### PR TITLE
Fix Key Naming Scheme in Raster Motion Task 

### PR DESCRIPTION
**Pull request by marrts**
Fix for #314

### CODE REVIEW

**Review by marip8 — APPROVED** [review]

**Comment by marrts** [comment]
The transition and rasters would have the same keys for getting the resulting trajectories which caused an issue with the final returned trajectory (demonstrated in #314). This fix ensures rasters and transitions keys have unique names. I think ideally these differences would come from the plugin yaml file, but trying to assign different names to all the keys breaks the planner entirely. Also, all the example task flow implementations have `”output_data”` as all the input and output keys, so I feel like the raster taskflow should be compatible with that, which this enables.

**Comment by Levi-Armstrong** [comment]
I see now the issue. When solve the rasters the results were stored in output_data_1 and when solving the transitions the results were stored in output_data_1 also. I think this is a reasonable long term fix for these two tasks.

**Comment by Levi-Armstrong** [comment]
@marrts Clang tidy build failed.

File: tesseract_task_composer/src/nodes/raster_motion_task.cpp [hunk 1]
```diff
@@ -64,7 +64,12 @@ createTask(const std::string& name,
   {
     std::map<std::string, std::string> input_renaming;
     for (const auto& x : input_indexing)
-      input_renaming[x] = task_name + "_" + x + std::to_string(index);
+    {
+      std::string name = task_name;
+      name.append("_");
+      name.append(std::to_string(index));
```
Comment (marrts) [inline tesseract_task_composer/src/nodes/raster_motion_task.cpp:70]: `x` was dropped from the name string in all of these

File: tesseract_task_composer/src/nodes/raster_only_motion_task.cpp [hunk 2]
```diff
@@ -81,8 +82,9 @@ createTask(const std::string& name,
     {
       std::string name = task_name;
       name.append("_");
+      name.append(x);
       name.append(std::to_string(index));
-      output_renaming[x] = name;
+      output_renaming[x] = task_name + "_" + x + std::to_string(index);
```
[the remark this answers is not in the document]
Comment (Levi-Armstrong) [reply tesseract_task_composer/src/nodes/raster_only_motion_task.cpp]: I am trying to get this in for a release I am doing and need to slow down.

**Pull request merged**